src/react/account/AccountList.jsx

```diff
@@ -0,0 +1,127 @@
+// @flow
+import React, { useEffect } from 'react';
+import Table, * as T from '../ui-elements/Table';
+import { useDispatch, useSelector } from 'react-redux';
+import { useFilters, useSortBy, useTable } from 'react-table';
+import type { Account } from '../../types/account';
+import type { AppState } from '../../types/state';
+import {IconCircle} from '../ui-elements/ListLayout';
+import { displayAccount } from '../actions';
+import { formatDate } from '../utils';
+import { push } from 'connected-react-router';
+import styled from 'styled-components';
+
+
+export const Icon = styled.i`
+    margin-left: 5px;
+`;
+
+const columns = [
+    {
+        Header: 'Name',
+        accessor: 'userName',
+    },
+    {
+        Header: 'Created on',
+        accessor: 'createDate',
+        Cell: ({ value }) => { return formatDate(new Date(value));},
+    },
+];
+
+const initialSortBy = [
+    {
+        id: 'createDate',
+        desc: true,
+    },
+];
+
+function AccountList() {
+    const dispatch = useDispatch();
+    const accountSelected = useSelector((state: AppState) => state.account.display);
+
+    // NOTE: accountList do not need to be memoized.
+    // "accountList"'s reference changes when a new configuration is set.
+    const accountList = useSelector((state: AppState) => state.configuration.latest.users);
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        rows,
+        setFilter,
+        prepareRow,
+    } = useTable({ columns, data: accountList, initialState: { sortBy: initialSortBy }, autoResetFilters: false, autoResetSortBy: false},
+        useFilters,
+        useSortBy,
+    );
+
+    useEffect(() => {
+        // NOTE: render the first/newest account after the mount (not on every render).
+        if (rows.length > 0) {
+            dispatch(displayAccount(rows[0].original));
+        }
+    }, []);
+
+    const rowClicked = (account: Account) => {
+        if (account.userName !== accountSelected.userName) {
+            dispatch(displayAccount(account));
+        }
+    };
+
+    const rowSelected = (accountName: string): boolean => {
+        return accountName === accountSelected.userName;
```

Comment:
I wonder if there can be a confusion in naming between account name and user name, since a user is technically an entity inside an account. In which case, could be good to rename `userName` to `accountName` if that's what it represents.